Lib/fontTools/varLib/__init__.py

```diff
@@ -313,7 +317,7 @@ def _add_gvar(font, model, master_ttfs, tolerance=0.5, optimize=True):
 		endPts = control[1] if control[0] >= 1 else list(range(len(control[1])))
 
 		for i,(delta,support) in enumerate(zip(deltas[1:], supports[1:])):
-			if all(abs(v) <= tolerance for v in delta.array):
+			if all(abs(v) <= tolerance for v in delta.array) and not isComposite:
 				continue
 			var = TupleVariation(support, delta)
 			if optimize:
```

Comment:
```suggestion
			if optimize and not isComposite:
```

We would have to skip the optimization as well to make sure the gvar record stays intact, but then it would be longer than necessary (0 for all points when just one point would be enough).